javascript/builtins/Number.json

```
@@ -952,7 +952,8 @@
                 "version_added": "18"
               },
               "edge": {
-                "version_added": "12"
+                "version_added": "12",
+                "notes": "In IE 11 and Edge 17 numbers are rounded to the 15 decimal digits: (1000000000000005).toLocaleString('en-US') gives '1,000,000,000,000,010'."
```

A couple things here. First, I did a little test and it seems to have been an issue for all the EdgeHTML releases. Also, we try to only mention the immediately relevant browser in notes. Plus some formatting suggestions.

```suggestion
                "notes": "Before Edge 79, numbers are rounded to 15 decimal digits. For example, <code>(1000000000000005).toLocaleString('en-US')</code> returns <code>\"1,000,000,000,000,010\"</code>."
```

---

@ddbeck , I cannot reproduce the bug on Microsoft EdgeHTML 18.18363 (Microsoft Edge 44.18362.449.0)